internal/pkg/deploy/cloudformation/stack/testdata/stacklocal/cf.params.json

```diff
@@ -10,7 +10,6 @@
     "HTTPSEnabled": "true",
     "LogRetention": "30",
     "RulePath": "my-svc",
```

Comment:
I thought in this PR we are removing these parameters (`RulePath`, `Stickiness`) - why are they still in this integration test 🤔 ? Is this test passing?

Reply:
+1 i think the tests pass because we don't remove them from the param list (which we might want to)

Reply:
I had understood the comment differently at the 1st place and that's why I only removed parameters from CFN templates and not param list which indeed fails the deployment and tests doesn't catch that for a reason that penghao mentioned above. So We need `RulePath`, `NLBPort` and `NLBAliases` in `uri.go` so we cannot get rid off them. But we can get rid off `Stickiness` so I am removing that one parameter only from the param list and cfn templates.

Reply:
We can probably get rid of these params by calling API to get the rule path, port, host header etc. But I think we can do that later.